app/views/application_view.rb

```diff
@@ -59,7 +59,7 @@ def head_links
   end
 
   def fb_campaigns_on
-    true
+    true unless Rails.env.test?
```

Comment:
What's this?

Reply:
this was to stop facebook errors in JS when running tests. I thought I had already removed it from this PR. Will do so again.

Reply:
in fact I think since https://github.com/europeana/Europeana-Patternlab/pull/245 has gone in we may be less likely to have facebook .js errors during testing at all.

...maybe.